global.json

```diff
@@ -1,12 +1,8 @@
 {
-  "sdk": {
```

Comment:
The file gets updated every time I run build from the root.

Reply:
Can you revert to what's in source? #586 will prevent this from happening again.